engine/base/kafka/src/main/scala/pl/touk/nussknacker/engine/baseengine/kafka/NuKafkaEngineApp.scala

```diff
@@ -0,0 +1,80 @@
+package pl.touk.nussknacker.engine.baseengine.kafka
+
+import com.typesafe.config.{Config, ConfigFactory}
+import org.apache.commons.io.FileUtils
+import pl.touk.nussknacker.engine.ModelData
+import pl.touk.nussknacker.engine.api.deployment.DeploymentData
+import pl.touk.nussknacker.engine.api.{JobData, ProcessVersion}
+import pl.touk.nussknacker.engine.baseengine.api.runtimecontext.EngineRuntimeContextPreparer
+import pl.touk.nussknacker.engine.baseengine.metrics.NoOpMetricsProvider
+import pl.touk.nussknacker.engine.canonize.ProcessCanonizer
+import pl.touk.nussknacker.engine.marshall.ProcessMarshaller
+import pl.touk.nussknacker.engine.util.config.CustomFicusInstances._
+import pl.touk.nussknacker.engine.util.loader.ModelClassLoader
+
+import java.io.File
+import java.lang.Thread.UncaughtExceptionHandler
+import java.net.URL
+import java.nio.file.Path
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.util.Using
+
+object NuKafkaEngineApp extends App {
+
+  val scenarioFileLocation = parseArgs
+
+  val scenario = parseScenario
+
+  val scenarioInterpreter = prepareScenarioInterpreter
+
+  Using.resource(scenarioInterpreter) { interpreter =>
+    interpreter.run()
+  }
+
+  private def parseArgs = {
+    if (args.length < 1) {
+      System.err.println("Missing scenario_file_location argument!")
```

Comment:
Do we want to add it also to normal log?

Reply:
not sure if it is a good approach. normal logs are also printed on stdout as it we also have in designer so we will have it duplicated

Reply:
Hmm... I started to look at our logback, and isn't it so, that we will have duplicate content in logs when using 'in background'? One in .out, one in .log?